### PR TITLE
dx-1210 Wrote gdpr disclaimer

### PR DESCRIPTION
**Pull request by arebra**
Wrote the include. Hasn't been included anywhere yet.

### CODE REVIEW

**Review by NullableInt — APPROVED** [review]
I like the wording 👍

File: _includes/alert-gdpr-disclaimer.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% include alert.html type="warning" icon="warning" body="GDPR sensitive data
```
Comment (payex-dev) [inline _includes/alert-gdpr-disclaimer.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6189093)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6189093&bid=20195037)

File: _includes/alert-gdpr-disclaimer.md (hunk 1 above)
Comment (arebra) [reply _includes/alert-gdpr-disclaimer.md:1]: @asbjornu What does this mean? :)

Comment (asbjornu) [reply _includes/alert-gdpr-disclaimer.md:1]: It's a bug in Codacy. Just ignore.

File: _includes/alert-gdpr-disclaimer.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+such as email, phone numbers and social security numbers must **not** be used as
+input in external parameters, such as `payerReference`. If it is necessary to
+use this kind of data, you must create a hash containing the information
+before using it as input." %}
```
Comment (asbjornu) [inline _includes/alert-gdpr-disclaimer.md:5]:
```suggestion
{% include alert.html type="warning" icon="warning" header="GDPR" body="GDPR
sensitive data such as email, phone numbers and social security numbers must
**not be used directly** in request fields such as `payerReference`. If it is
necessary to use GDPR sensitive data, it must be hashed and then the hash can be
used in requests towards Swedbank Pay." %}

```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by asbjornu — APPROVED** [review]

**Pull request merged**